### PR TITLE
Created Reviewer and Industries blueprint and routes

### PR DESCRIPTION
**Pull request by TranQv54**
Created routes for reviewer and industries from API matrix. Removed num_posts data values from 01_glasshusky_data since it's a derived attribute.

### CODE REVIEW

File: api/backend/reviewers/reviewers_routes.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+# routes.
+reviewers = Blueprint('reviewers', __name__)
+
+#------------------------------------------------------------
+# Get all the reviewers from the database, package them up,
+# and return them to the client
+@reviewers.route('/reviewers', methods=['GET'])
+def get_reviewers():
+    query = '''
+        SELECT major, name, num_co-ops, year, num_posts, bio
```
Comment (karynayen) [inline api/backend/reviewers/reviewers_routes.py]: I feel like it might be good to just return all data (SELECT *)

File: api/backend/reviewers/reviewers_routes.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    response.status_code = 200
+    # send the response back to the client
+    return response
+
+
+# ------------------------------------------------------------
+# This is a POST route to add a new product.
+# Remember, we are using POST routes to create new entries
+# in the database. 
+@reviewers.route('/reviewers/<reviewerID>', methods=['PUT'])
```
Comment (karynayen) [inline api/backend/reviewers/reviewers_routes.py]: Whoops, I didn't notice this when I was dividing the tasks but we should probably have both a POST and PUT route. 
It looks like here you are defining the POST route. 
By convention, if you are adding an object to a db, you don't have a reviewID yet so the route would be '/reviewers'. 
Additionally, the methods should be ['POST'] instead of 'PUT'

Can you also implement a route to update an existing reviewer?
@reviewers.route('/reviewers/<reviewerID>', methods=['PUT'])

**Comment by TranQv54** [comment]
Uhhh I think I did this right? I merged karyna's from the company-routes branch into mine (Vinh) since it hadn't merged into the main yet. Then I wrote my code so hopefully there aren't any issues.

File: app/src/pages/12_API_Test.py [hunk 3]
```diff
@@ -17,7 +17,7 @@
 """
 data = {} 
 try:
-  data = requests.get('http://api:4000/data').json()
+  data = requests.get('http://api:4000/i/industries/3/companies').json()
```
Comment (karynayen) [inline app/src/pages/12_API_Test.py]: not a huge deal but you should probably undo this change

File: api/backend/industries/industries_routes.py [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+    cursor.execute(query)
+    db.get_db().commit()
+    
+    response = make_response("Successfully added product")
+    response.status_code = 200
+    return response
+
+
+
+# # ------------------------------------------------------------
```
Comment (karynayen) [inline api/backend/industries/industries_routes.py]: please delete extra comments

**Review by karynayen — CHANGES_REQUESTED** [review]
small changes requested

**Review by karynayen — APPROVED** [review]
LGTM
